README.md

```diff
@@ -143,11 +170,11 @@ We can generate a strings.xml file for Android with the English translations pro
 
 A similar file with the Dutch translations will be created as well if we request localicious to do so:
 
-`localicious render ./copy.yaml ./output_path --platforms android --languages en,nl`
+`localicious render ./copy.yaml ./output_path --outputTypes android --collections ANDROID,SHARED --languages en,nl`
 
-By changing the destination, like so:
+By changing the destinatiooutput type, like so:
 
-`localicious render ./copy.yaml ./output_path --platforms ios --languages en`
+`localicious render ./copy.yaml ./output_path --outputTypes ios --collections IOS,SHARED --languages en`
```

Comment:
After writing [this](https://github.com/PicnicSupermarket/localicious/pull/41/files#r458894429), I discovered that I've used inconsistent casing here as well... 
I'll leave it until we decide which way to go :)

Reply:
Also, we assume collection names with all caps.

`./bin/localicious render ./copy.yaml ./output_path --outputTypes android,ios --collections ANDROID,Shared --languages en,nl`

With a collection named `Shared`
```
Shared:
  Delivery:
    Widget:
```

Is not supported.

```
❌ Something went wrong rendering.
❌ Your Localicipe contains validation issues:
    -  should have required property 'SHARED'
```

But I think the error message is clear enough. I would not change